vm/error_test.go

```diff
@@ -159,33 +161,33 @@ func TestStackTraces(t *testing.T) {
 
 func TestNoMethodError(t *testing.T) {
 	tests := []errorTestCase{
-		{`a`, "NoMethodError: Undefined Method 'a' for <Instance of: Object>", 1},
+		{`a`, "NoMethodError: Undefined Method 'a' for #<Object:XXXXXXXXXXXX >", 1},
```

Comment:
Think we should also add a case for class methods (`Foo.bar`). Can you add it?

Reply:
OK, I'm now adding more tests to the PR so I'll include the cases.